Services/Curl.php

```diff
@@ -95,7 +95,9 @@ public function sendRequest($url, $method, array $options = array(), $payload =
         $curlMetaData = (object) curl_getinfo($this->curl);
 
         $this->curlOptionsHandler->reset();
-        function_exists(curl_reset($this->curl)) ? curl_reset($this->curl) : curl_setopt_array($this->curl, $this->curlOptionsHandler->getOptions());
+        if (function_exists('curl_reset')) {
```

Comment:
There is no reset functionality for the case the "curl_reset" does not exist. Use the ternary operator like suggested, excepting the wrong usage of function_exists.